3.7/config.sh

```diff
@@ -35,3 +35,7 @@ if [ ! -z "$IMPORT_STYLE" ]; then
 else
   sed -i "s|__IMPORT_STYLE__|full|g" ${CONFIG_FILE}
 fi
+
+# if flatnode directory was created by volume / mount, use flatnode files
+
+if [ -d "${PROJECT_DIR}/flatnode" ]; then sed -i 's\NOMINATIM_FLATNODE_FILE=\NOMINATIM_FLATNODE_FILE="/nominatim/flatnode/flatnode.file"\g' ${CONFIG_FILE}; fi
```

Comment:
The documentation is unclear about this: should you keep the file around after import?

Reply:
If you plan to update your data, yes.

Reply:
What I can try: import with flatnode, start container without mount, change setting to disable flatnode and update the data then. Will check and maybe add that change here in addition to catch that case.

@mtmail is there anything that would speak against using a flatnode file only for the initial import?

Reply:
BTW, it's not a requirement for me for this feature to be idiot-proof. If you want to use it, you should know what to do.

Reply:
True, but maybe it's a viable option I didn't think about before that saves space on your disk.